### ISSUE TITLE
[ForceGenerator] ComStar and WoB can't generate mech only Level II's

### ISSUE THREAD

**Issue by repligator** [MegaMek/megamek#5666]
### Environment
Linux
Java 17
Developmental

### Description
ComStar and World of Blake are limited to a single Unit Type in Force Generator; Combined. This prevents the force generator from generating mech only Level II's for these factions.

**Comment by HammerGS** [comment]
This looks intentional per the notes.

!-- The ComGuards use combined arms at the LIII (demi-company) level
    and above. According to FM:CS LIIs contain a single unit type
    "in many cases" but combined-arms units are possible. FM:CS gives
    the approximate numbers of each unit type in LIVs and notes that
    the ratios are similar in LIIIs. Each LIV has 39 ASFs, which would
    result in each LIII getting a squadron with three left over. ASFs
    are deployed as flights rather than grouped together, so each LIII
    would have three LIIs containing two ASFs and four LIIs of ground
    units and three LIIs of ground units only. I have assumed that the
    remaining three are combined with three Meks to form the LIV CO's
    command LII. To limit the remaining combinations to a reasonable
    level, I have assumed also that any combined arms LII has at least
    two of any unit type it contains, and units are grouped together as
    much as possible (e.g. six Meks and six tanks will always be a Mek
    LII and a tank LII, rather than two 3/3s or a 4/2 and a 2/4).
    
    Nearly all the LIVs on the TO&E have two LIIIs with the same alphabetic
    designation and one each of the next two closest before and after
    (e.g. alpha, beta, gamma, gamma, delta, epsilon). Other arrangements
    show up, but trying to account for those would add more complexity
    than it's worth. -->

**Comment by repligator** [comment]
Yes, I read that as 'Levels II _may_ be combined arms' rather than 'Level II _must_ be combined arms'. I don't own FM:CS so I can't really say one way or the other. Apologies if I'm misunderstanding.
